### PR TITLE
Whitelist in package.json only required files

### PR DESCRIPTION
**Pull request by alobaidizt**
Part of the quest to reduce the number of files of ember-cli: ember-cli/ember-cli#6147

Removing `.npmignore` is recommended after merging this PR. One less file to manage. 😄

### CODE REVIEW

**Comment by jonkoops** [comment]
@alobaidizt You could probably also remove .npmignore in this pull request instead of someone having to do it afterwards.

**Comment by stefanpenner** [comment]
@alobaidizt  mind updating ^

**Comment by alobaidizt** [comment]
@jonkoops @stefanpenner Updated!

File: package.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
     "start": "ember server",
     "test": "ember try:testall"
   },
+  "files": [
+    "config/environment.js",
```
Comment (stefanpenner) [inline package.json]: why is this included?

**Comment by alobaidizt** [comment]
@stefanpenner should be good to go now.

**Comment by stefanpenner** [comment]
thanks, released as 2.1.1
